<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="imgLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="editorWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>260</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="gist_wdg">
       <attribute name="title">
        <string>Гистограмма</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>831</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="PlotWidget" name="all_gist"/>
         </item>
         <item>
          <widget class="PlotWidget" name="red_gist"/>
         </item>
         <item>
          <widget class="PlotWidget" name="blue_gist"/>
         </item>
         <item>
          <widget class="PlotWidget" name="green_gist"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="rgb_wdg">
       <attribute name="title">
        <string>RGB</string>
       </attribute>
       <widget class="QSlider" name="red_sld">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>641</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="green_sld">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>641</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="blue_sld">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>130</y>
          <width>641</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>121</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>RED</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>121</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>GREEN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>121</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>BLUE</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="resize_wdj">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QLineEdit" name="width_edit">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>60</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="height_edit">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>110</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>60</y>
          <width>121</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Ширина</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>110</y>
          <width>121</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Высота</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="edit_wdg">
       <attribute name="title">
        <string>Редактировать</string>
       </attribute>
       <widget class="QSlider" name="brightness_sld">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>641</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="contrast_sld">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>130</y>
          <width>641</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>121</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Яркость</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>131</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Констрастность</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reset_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uploadBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Загрузить</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>editorWidget</zorder>
   <zorder>imgLabel</zorder>
   <zorder>uploadBtn</zorder>
   <zorder>reset_btn</zorder>
   <zorder>saveBtn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
